modules/wp-cli/library.php

```diff
@@ -4,6 +4,7 @@
 use Elementor\Api;
 use Elementor\Plugin;
 use Elementor\TemplateLibrary\Source_Local;
+use ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Cache;
```

Comment:
you can't use elementor pro here..
you inside elementor core ( they don't know each other )

Reply:
Moving this cli command to pro